proto/sifnode/ethbridge/v1/types.proto

```diff
@@ -62,4 +62,9 @@ enum ClaimType {
 }
 
 // GenesisState for ethbridge
-message GenesisState { string crosschain_fee_receive_account = 1; }
+message GenesisState { 
```

Comment:
CrosschainFeeReceiveAccount already checked, for other three items, they are map, so that they are empty is correct if no any data before in peggy1.0. Do you mean we should check each entry in these maps?